ee/maintained-apps/README.md

```diff
@@ -66,3 +66,23 @@ These are command lines that will be run _before_ the generated uninstall script
 
 #### `post_uninstall_scripts`
 These are command lines that will be run _after_ the generated uninstall script is executed.
+
+### Testing
+
+Fleet tests every change to supported Fleet-maintained apps (e.g. new version, updated install script, etc.) and all new apps.
+
+For new apps:
+
+1. When a pull request is opened in `inputs/`, the [#g-software Engineering Manager (EM)](https://fleetdm.com/handbook/company/product-groups#software-group) is automatically added as reviewer.
+2. The EM is responsible for making sure that the `name` for the new app matches the name that shows up in Fleet's software inventory. If the name doesn't match or if the name is not user-friendly, the EM will bring it to #g-software design review. This way, when the app is added to Fleet, the app will be matched with the app that comes back in software inventory.
+3. Then, the EM merges the PR. This automatically opens a PR to `outputs/`
+4. Next, follow steps under "For changes to supported apps." 
+
+For changes to supported apps:
+
+1. A pull request (PR) is automatically opened in `outputs/` and a Product Designer is automatically added as reviwer.
+2. The Product Designer is responsible for testing the app:
+- If the app is a new app, add an icon for the app to the PR. App icons are located [here in the repository](https://github.com/fleetdm/fleet/tree/main/frontend/pages/SoftwarePage/components/icons).
```

Comment:
Unless you rebuild Fleet, the icon changes won't be reflected (and new app icons will only be released on subsequent Fleet server releases) due to desciping of work that was going to be in FMAv2. To cut coordination requirements, we should bring that work back in a new ticket.

Reply:
Sounds good. Please feel free to open an engineering-initiated request.

Reply:
Here's what we're thinking in the meantime:

If the test is successful, the PD approves and merges the PR. For new apps, the PD opens a user story to test the new icon and assigns the EM. The EM brings this story onto the release board.